src/extensions/TableHeaderImproved.ts

```diff
@@ -0,0 +1,26 @@
+import { TableHeader } from "@tiptap/extension-table-header";
+
+const TableHeaderImproved = TableHeader.extend({
```

Comment:
We'd want to document the new extensions and whatnot in the README if we're going to merge this